roles/container-engine/containerd/defaults/main.yml

```diff
@@ -46,7 +46,13 @@ containerd_base_runtime_specs:
 containerd_grpc_max_recv_message_size: 16777216
 containerd_grpc_max_send_message_size: 16777216
 
-containerd_debug_level: "info"
+# debug socket location: unix or tcp format
+containerd_debug_address: ""
+containerd_debug_level: ""
+# logs format, supported values: text, json
+containerd_debug_format: ""
+containerd_debug_uid: 0
+containerd_debug_gid: 0
```

Comment:
I'd rather have some consistency here : either put the default for all settings, or for none.
And I'd lean towards documenting the defaults explicitely.

Reply:
I put it to the `containerd config default` as requested previously, I changed it back to what it was for `containerd_debug_level` and add the others only if they are defined by the user